.circleci/config.yml

```diff
@@ -361,7 +361,7 @@ jobs:
   ruby-checks:
     docker:
       - image: cimg/ruby:3.2.2
-    resource_class: small
+    resource_class: medium
```

Comment:
Do we need it to be medium?

Reply:
Yes it kept dying when checking the files before and even tried it here in a separate PR and still got the kill signal from CI

Reply:
Its hitting memory limits otherwise - probably number of changed files so whilst this is a bit PR, going forward it'll be scanning all files anyway so running out of memory will become an common issue

This step runs for all of 20 seconds so the impact on cost is tiny